py/htm/advanced/regions/ApicalTMPairRegion.py

```diff
@@ -522,3 +522,12 @@ def getAlgorithmInstance(self):
         Returns an instance of the underlying temporal memory instance
         """
         return self._tm
+
+    def saveConnectionsToFile(self, filepath):
```

Comment:
why not using the `executeCommand` here as well?

Reply:
This method is called by executeCommand.

Python implemented regions works like following: if you call executeCommand("whatever") on pyRegion, it natively tries to call method "whatever" of pythonRegion implementation (firstly checks if that attribute exists).

I think this is nice property of these python regions.

Reply:
> executeCommand("whatever") on pyRegion, it natively tries to call method "whatever" of pythonRegion 

oh, I see. I've seen you implementing the "saveConnectionsToFile" in the c++ SPRegion. Didn't know it works differently for PyRegions. thank you for explanation. 
Btw, through some reflection we could implement the same approach for cpp regions, I think? But may not be worth the problems as the if-else in executeCommand in cpp works well.

Reply:
> Btw, through some reflection we could implement the same approach for cpp regions, I think? But may not be worth the problems as the if-else in executeCommand in cpp works well.

Maybe possible, but as you say, for c++ i would keep as-is